src/modules/commander/commander_params.c

```diff
@@ -802,6 +802,21 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
  * Temporary Parameter to enable interface testing
```

Comment:
Still temporary?

Reply:
Good catch :smile:, guess I can get rid of that comment.